src/o365/spfx/commands/project/project-upgrade.ts

```diff
@@ -488,6 +507,78 @@ ${f.resolution}
     return s.join('').trim();
   }
 
+  private getTourReport(findings: FindingToReport[], project: Project): string {
+    const tourFindings: FindingTour = {
+      title: `Upgrade project ${path.posix.basename(this.projectRootPath as string)} to v${this.toVersion}`,
+      steps: []
+    };
+
+    findings.forEach(f => {
+      const lineNumber: number = f.position && f.position.line ? f.position.line : this.getLineToModify(f, project.path);
+
+      let resolution: string = '';
+      switch (f.resolutionType) {
+        case 'cmd':
+          resolution = `Execute the following command:\r\n>> ${f.resolution}`;
```

Comment:
For consistency, we should markup commands to execute as code with `sh` (shell) for language

Reply:
I'm not sure what you mean with this one. In the CodeTour schema, using `>>` embeds a shell command that can be clicked to launch in the terminal.